siad/main.go

```diff
@@ -168,6 +170,7 @@ func main() {
 	root.Flags().StringVarP(&globalConfig.Siad.Modules, "modules", "M", "cghrtw", "enabled modules, see 'siad modules' for more info")
 	root.Flags().BoolVarP(&globalConfig.Siad.AuthenticateAPI, "authenticate-api", "", false, "enable API password protection")
 	root.Flags().BoolVarP(&globalConfig.Siad.AllowAPIBind, "disable-api-security", "", false, "allow siad to listen on a non-localhost address (DANGEROUS)")
+	root.Flags().BoolVarP(&globalConfig.Siad.MemLogging, "enable-memlogging", "", false, "enable the logging of the sizes of various module data structures")
```

Comment:
*should* this be in the API? Is there a technical reason why it can't be just a `siad` flag? That makes more sense to me than adding a debugging hook to the public API.